plugins/api-docs/src/components/ApisCards/presets.tsx

```diff
@@ -22,10 +22,14 @@ import ToggleButton from '@material-ui/lab/ToggleButton';
 import React, { useState } from 'react';
 import { ApiTypeTitle } from '../ApiDefinitionCard';
 import { ApiDefinitionDialog } from '../ApiDefinitionDialog';
+import { TranslationFunction } from '@backstage/core-plugin-api/alpha';
+import { apiDocsTranslationRef } from '../../translation';
 
-export function createSpecApiTypeColumn(): TableColumn<ApiEntity> {
+export function createSpecApiTypeColumn(
+  t: TranslationFunction<typeof apiDocsTranslationRef.T>,
+): TableColumn<ApiEntity> {
   return {
-    title: 'Type',
+    title: t('apiEntityColumns.typeTitle'),
     field: 'spec.type',
     render: entity => <ApiTypeTitle apiEntity={entity} />,
   };
```

Comment:
Could you please translate the toggle buttno aria label?

Reply:
sure